src/travel-details-screens/DepartureDetailsScreenComponent.tsx

```diff
@@ -126,9 +126,9 @@ export const DepartureDetailsScreenComponent = ({
                 {title ?? t(DepartureDetailsTexts.header.notFound)}
               </ThemeText>
             </View>
-            {lastPassedStop || (vehiclePosition && mapData) ? (
+            {realtimeText || (vehiclePosition && mapData) ? (
               <View style={styles.headerSubSection}>
```

Comment:
I think we need to change the logic here when the map is removed. If there is no vehicle location data, there should still be a "Show in map" button. (Meaning, this section will always be visible)